### PR TITLE
update

### PR DESCRIPTION
**Pull request by Luciker**
实践踩过的坑

### CODE REVIEW

**Comment by Luciker** [comment]
test

File: IMPORTANT.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+1、Python\Lib\site-packages\中的crypt改成Crypto
+2、pip install pycrypto
+3、不行就pip uninstall Crypto 
+        pip uninstall pycrypto
+        pip install Crypto
+        pip install pycrypto
+    然后重复1 2
```
Comment (antonioCoco) [inline IMPORTANT.md]: You don't need to uninstall libraries, you need just to :
pip install -r requirements.txt
Tested on all latest version of ubuntu,debian,kali and it's working

File: IMPORTANT.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+4、注释掉core\SharPyShellPrompt.py的33行
+ #signal.signal(signal.SIGTSTP, lambda s, f: self.do_quit())
```
Comment (antonioCoco) [inline IMPORTANT.md]: if you comment out this line when you are in the terminal and press ctrl+z you will leave traces on the target system.

**Comment by antonioCoco** [comment]
Added comments on the code

**Pull request closed without merging**